src/travis2docker/templates/build.sh

```diff
@@ -168,6 +168,108 @@ chown_all(){
     chown -R odoo:odoo /etc/postgresql-common/common-vauxoo.conf
 }
 
+configure_vim(){
+    if [ -z ${VIM_INSTALL+x} ];
+    then
+      return 0;
+    fi
+    # Upgrade & configure vim
+    apt install vim --only-upgrade
+    # Get vim version
+    VIM_VERSION=$(dpkg -s vim | grep Version | sed -n 's/.*\([0-9]\+\.[0-9]\+\)\..*/\1/p' | sed -r 's/\.//g')
+
+    wget -q -O /usr/share/vim/vim${VIM_VERSION}/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
+    git_clone_execute "${SPF13_REPO}" "3.0" "bootstrap.sh"
```

Comment:
This method doesn't exists in the new quay.io image

I think the entire method needs to be refactored in order to be compatible with it

Try using a quay.io image to test it, please

Reply:
It is only cloning and executing a script cloned

This time I think we don't need an extra method to do it since it could be great copy this method and run locally without containers and be compatible